### PR TITLE
APIGOV-16970 - Update page in Connect AWS Gateway “deploy-your-agents-with-amplify-cli” 

### PR DESCRIPTION
**Pull request by kflan-io**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by kflan-io** [comment]
@dfeldick @jcollins-axway @sbolosan

File: content/en/docs/central/connect-aws-gateway/deploy-your-agents-with-amplify-cli.md [hunk 1]
```diff
@@ -107,15 +105,15 @@ The installation procedure prompts for the following:
...
+   * Traceability Agent protocol (Lumberjack (tcp) by default), select between `Lumberjack`, or `HTTPS`
 
 Once you have answered all questions, the cloud formation templates are downloaded and pre-configured, the agents' configuration files are updated, the Amplify Central resources are created and the key pair is generated (if you chose to create a new service account).
 
 The current directory contains the following files:
 
 ```shell
-da_env_vars.env                   *EC2 Deployment Only
-ta_env_vars.env                   *EC2 Deployment Only
+da_env_vars.env                   *EC2 / 'Docker Container Only' Deployments Only
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/deploy-your-agents-with-amplify-cli.md:115]: 115 and 116... its confusing.  Are you trying to say *EC2 or Docker Container Deployments Only ?

**Review by sbolosan — APPROVED** [review]

File: content/en/docs/central/connect-aws-gateway/deploy-your-agents-with-amplify-cli.md (hunk 1 above)
Comment (kflan-io) [reply content/en/docs/central/connect-aws-gateway/deploy-your-agents-with-amplify-cli.md]: EC2 and Docker Container Deployments Only, it is.

**Review by lbadenhop — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes
@lbadenhop please proceed

**Pull request merged**